libs/testdiff/golden.go

```diff
@@ -17,6 +16,11 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+const (
+	testerName      = "$USERNAME"
+	testerWithEmail = testerName
```

Comment:
We can get rid of this constant as well right? Why have two constants?

Reply:
fair point, removed one.